util/fmtutil/format_test.go

```diff
@@ -80,17 +82,17 @@ var writeRequestFixture = &prompb.WriteRequest{
 		},
 		{
 			Labels: []prompb.Label{
-				{Name: "__name__", Value: "http_request_duration_seconds_sum"},
+				{Name: "__name__", Value: "http_request_duration_seconds_count"},
 				{Name: "job", Value: "promtool"},
 			},
-			Samples: []prompb.Sample{{Value: 53423, Timestamp: 1}},
+			Samples: []prompb.Sample{{Value: 144320, Timestamp: 1}},
 		},
 		{
 			Labels: []prompb.Label{
-				{Name: "__name__", Value: "http_request_duration_seconds_count"},
+				{Name: "__name__", Value: "http_request_duration_seconds_sum"},
 				{Name: "job", Value: "promtool"},
 			},
-			Samples: []prompb.Sample{{Value: 144320, Timestamp: 1}},
+			Samples: []prompb.Sample{{Value: 53423, Timestamp: 1}},
```

Comment:
I'm not 100% sure why this is changing.

The output is supposed to be ordered by metric name, so bucket comes before count before sum. The order _should_ be the order shown here, and I don't think I'm actually changing something to modify that...

Reply:
The old code simply used the "wrong" order. (I don't think that the remote-write spec requires this kind of ordering, but it cannot harm to do it your way.) tl;dr: This is fine.

Reply:
Yes there is no remote write spec about this ordering, I was just following this rule : label names sorted in lexicographical order.